app/models/server_role.rb

```diff
@@ -5,32 +5,19 @@ class ServerRole < ActiveRecord::Base
   validates_presence_of     :name
   validates_uniqueness_of   :name
 
-  FIXTURE_DIR = File.join(Rails.root, "db/fixtures")
-
-  def self.seed_data
-    File.read(File.join(FIXTURE_DIR, "#{to_s.pluralize.underscore}.csv"))
-  end
-
   def self.seed
-    data = seed_data.split("\n")
-    cols = data.shift.split(",")
-
-    data.each do |a|
-      next if a =~ /^#.*$/ # skip commented lines
+    CSV.foreach(fixture_path, :headers => true, :skip_lines => /^#/).each do |csv_row|
+      action = csv_row.to_hash
+      action['license_required'] ||= ''
```

Comment:
There is an existing test that explicitly checks if it is a blank line. I'm not sure if it is _really_ needed, so I added this line in order to preserve existing behaviour. If we don't need it - I'll remove the line and update the test accordingly.